torchio/data/image.py

```diff
@@ -647,6 +647,42 @@ def plot(self, **kwargs) -> None:
             from ..visualization import plot_volume  # avoid circular import
             plot_volume(self, **kwargs)
 
+    def show(self, viewer_path: Optional[TypePath] = None) -> None:
+        """Open the image using external software.
+
+        Args:
+            viewer_path: Path to the application used to view the image. If
+                ``None``, the value of the environment variable
+                ``SITK_SHOW_COMMAND`` will be used. If this variable is also
+                not set, TorchIO will try to guess the location of
+                `ITK-SNAP <http://www.itksnap.org/pmwiki/pmwiki.php>`_ and
+                `3D Slicer <https://www.slicer.org/>`_.
+
+        Raises:
+            RuntimeError: If the viewer is not found.
+        """
+        sitk_image = self.as_sitk()
+        image_viewer = sitk.ImageViewer()
+        # This is so that 3D Slicer creates segmentation nodes from label maps
+        if self.__class__.__name__ == 'LabelMap':
+            print('Im here')
```

Comment:
`print('Not for long!')`

Reply:
Hahahah `pre-commit` should have detected that and not allowed me to commit. There's obviously something wrong so this is a good catch. Thanks!